src/mongoc/mongoc-bulk-operation.h

```diff
@@ -76,7 +76,7 @@ void                     mongoc_bulk_operation_set_client        (mongoc_bulk_op
                                                                   void                         *client);
 void                     mongoc_bulk_operation_set_hint          (mongoc_bulk_operation_t      *bulk,
                                                                   uint32_t                      hint);
```

Comment:
Sorry for the ASCII art changes but can we line up the above function parameters with those of mongoc_bulk_operation_get_write_concern's below?